src/work-allocation/containers/task-manager-list/task-manager-list.component.ts

```diff
@@ -66,11 +66,6 @@ export class TaskManagerListComponent extends TaskListWrapperComponent implement
   public ngOnInit(): void {
     super.ngOnInit();
     // Get the caseworkers and locations for this component.
-    this.caseworkerService.getAll().subscribe(caseworkers => {
```

Comment:
Seeing as `this.caseworkers` is now being set in the `TaskListWrapperComponent` instead, I think the `caseworkers` declaration within this component should be removed as it's redundant and confusing (or it may even cause an issue).

Reply:
Agree, I thought i removed it.

Reply:
Its done now.